Services/WebWorker/WorkerHost.cpp

```diff
@@ -68,7 +68,7 @@ void DedicatedWorkerHost::run(GC::Ref<Web::Page> page, Web::HTML::TransferDataHo
 
     // 10. Set worker global scope's name to the value of options's name member.
     if (is_shared)
-        TODO();
+        static_cast<Web::HTML::SharedWorkerGlobalScope&>(*worker_global_scope).set_name(m_name);
     else
```

Comment:
One very minor nit I just noticed (otherwise all looks good to me), but since this is in `WorkerGlobalScope` now we can remove the branch